fio/fio_run

```diff
@@ -845,46 +848,45 @@ execute_test()
 	results_prefix=$additional_info"user_"$to_puser"_instance_"$to_sysname"_numb_disks_"$numb_disks"_"$tuning_info
 	user=$additional_info"user_"$to_user"_instance_"$to_sysname"_numb_disks_"$numb_disks
 	pushd $working_dir > /dev/null
-	if [[ ! -f results_fio.csv ]]; then
-		echo Failed, results_fio.csv missing >> test_results_report
-	else
-		value=`head -2 results_fio.csv | tail -1`
-		jobs=`echo $value | cut -d':' -f 1`
-		tput=`echo $value | cut -d':' -f 4 | cut -d'.' -f 1`
-
-		if [[ $jobs -gt 0 ]] && [[ $tput -gt 0 ]]; then
-			echo Ran >> test_results_report
-		else
-			echo Failed >> test_results_report
-		fi
-	fi
 	popd > /dev/null
 
 	if [ $to_pbench -eq 1 ]; then
 		cd /var/lib/pbench-agent/
-		mkdir export_fio_data
-		cp ${curdir}/meta_data* export_fio_data
+		mkdir ${curdir}/export_fio_data
+		cp ${curdir}/meta_data* ${curdir}/export_fio_data
 		ls -d pbench-user* > /dev/null
 		if [ $? -ne 0 ]; then
 			for i in `ls -d fio_*`; do
-				mkdir export_fio_data/$i
+				mkdir ${curdir}/export_fio_data/$i
 				cp ${curdir}/meta_data*yml $i
-				cp $i/result.csv export_fio_data/$i
-				cp $i/result.txt export_fio_data/$i
+				cp $i/result.csv ${curdir}/export_fio_data/$i
+				cp $i/result.txt ${curdir}/export_fio_data/$i
 			done
 		else
 			for i in `ls -d pbench-user*`; do
-				mkdir export_fio_data/$i
+				mkdir ${curdir}/export_fio_data/$i
 				cp ${curdir}/meta_data*yml $i
-				cp $i/result.csv export_fio_data/$i
-				cp $i/result.txt export_fio_data/$i
+				cp $i/result.csv ${curdir}/export_fio_data/$i
+				cp $i/result.txt ${curdir}/export_fio_data/$i
 			done
 		fi
+		if [ $use_pbench_version -eq 1 ]; then
+			pushd  ${curdir}/export_fio_data > /dev/null
```

Comment:
These redirects are a little discomforting.  I'm not going to hold it up, but instead of pushd/popd perhaps we should just use `cd <dir>` and `cd -`.  Now get off my lawn.

Reply:
Changed these 2.  We get arid of these for the most part when we move away from pbench storing data for us.  The data from the test appearing in two different locations, along with the extra stuff pbench generates is causing this.